hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxySupport.java

```diff
@@ -972,10 +972,6 @@ public void executeOnEntriesInternal(EntryProcessor entryProcessor, Predicate pr
         return serializationService.toObject(obj);
     }
 
-    protected Data toData(Object obj) {
```

Comment:
Should this be part of this PR. Looks like a non-related cleanup (or I'm missing something).